vertical-pod-autoscaler/pkg/recommender/model/cluster.go

```diff
@@ -246,7 +246,10 @@ func (cluster *ClusterState) RecordOOM(containerID ContainerID, timestamp time.T
 // all aggregations it matches.
 func (cluster *ClusterState) AddOrUpdateVpa(apiObject *vpa_types.VerticalPodAutoscaler, selector labels.Selector) error {
 	vpaID := VpaID{Namespace: apiObject.Namespace, VpaName: apiObject.Name}
-	annotationsMap := apiObject.Annotations
+	annotationsMap := make(vpaAnnotationsMap)
+	for key, value := range apiObject.Annotations {
+		annotationsMap[key] = value
```

Comment:
OOC, why was this change required?

Reply:
Thanks for the catch - it was a remnant of the old oom timestamp idea. Removed it